### PR TITLE
Add a basic bqplot exercise to the bqplot notebook

### PR DESCRIPTION
**Pull request by ibdafna**

### CODE REVIEW

**Comment by ibdafna** [comment]
Oops, the ipyleaflet version fix commit is still here. We may still be able to merge..I'll reopen if not

**Comment by martinRenou** [comment]
> Oops, the ipyleaflet version fix commit is still here. We may still be able to merge..I'll reopen if not

It will disappear if you rebase your branch:
```
git checkout master && git pull upstream master
git checkout add_bqplot_exercises
git rebase -i master
git push upstream add_bqplot_exercises
```

File: notebooks/07.03-bqplot.ipynb [hunk 1]
```diff
@@ -379,13 +379,48 @@
...
+   "outputs": [],
+   "source": []
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "widgets-tutorial",
+   "display_name": "Python 3",
    "language": "python",
-   "name": "widgets-tutorial"
```
Comment (martinRenou) [inline notebooks/07.03-bqplot.ipynb:388]: Maybe you should not have committed the changes in the kernel name. But that is detail I guess.

Comment (martinRenou) [reply notebooks/07.03-bqplot.ipynb:388]: Let's ignore this

**Pull request merged**